benchmark/unified_benchmark.cpp

```diff
@@ -210,104 +210,104 @@ BenchmarkResult run_benchmark(const std::string& name,
                 return result;
             }
         }
-        
+
         // Timed run
         auto start = std::chrono::steady_clock::now();
-        
+
         for (int i = 0; i < iterations; i++) {
             if (!f()) {
                 result.success = false;
                 return result;
             }
         }
-        
+
         auto end = std::chrono::steady_clock::now();
         auto duration = std::chrono::duration_cast<std::chrono::microseconds>(end - start);
-        
+
         result.time_per_iteration_us = duration.count() / (double)iterations;
-        result.throughput_mbps = (json.size() * iterations / 1024.0 / 1024.0) / 
+        result.throughput_mbps = (json.size() * iterations / 1024.0 / 1024.0) /
                                 (duration.count() / 1000000.0);
         result.success = true;
     } catch (...) {
         result.success = false;
     }
-    
+
     return result;
 }
 
 // Twitter benchmarks
 std::vector<BenchmarkResult> benchmark_twitter_parsing(const std::string& json) {
     std::vector<BenchmarkResult> results;
-    const int iterations = 1000;
-    
+    const int iterations = 3000;
+
     // 1. simdjson - Manual (conservative approach - fresh parser each iteration)
     {
         padded_string padded(json); // Create once
-        
+
         auto benchmark = [&]() -> bool {
             try {
-                ondemand::parser parser; // Fresh parser each iteration
+                auto &parser = ondemand::parser::get_parser();
```

Comment:
Makes sense to grab a new instance of the global parser. Good catch!

Reply:
@FranciscoThiesen It actually does not matter as far as performance goes in this instance but the principle in simdjson is to reuse the parser.